utilities/cluster_config_utils.py

```diff
@@ -186,6 +186,20 @@ def is_ipv6(cluster_config):
     return cluster["environment"]["ipv6_enabled"]
 
 
+def get_cbs_primary_nodes_str(cluster_config, cbs_nodes):
+    if is_ipv6(cluster_config):
```

Comment:
could you please add some comments to explain what scenario this code is dealing with or point to a sample cluster_config file for easy understanding? is_ipv6 is a flag in [environment] section, which suppose to be global (for all cbs nodes) I assume?

Reply:
Added the summary